webapp/app/css/treeherder.css

```diff
@@ -972,6 +972,17 @@ ul.failure-summary-list li .btn-xs {
     margin-bottom: -1px;
 }
 
+.pinboard-related-bugs-btn > .pinboard-related-bugs-link,
+.pinboard-related-bugs-btn:hover > .pinboard-related-bugs-link:hover {
+    border-top-right-radius: 3px !important;
+    border-bottom-right-radius: 3px !important;
+}
+
+.pinboard-related-bugs-btn:hover > .pinboard-related-bugs-link {
+    border-top-right-radius: 0 !important;
+    border-bottom-right-radius: 0 !important;
+}
```

Comment:
I checked again, this aims to overwrite a bootstrap rule `.btn-group > .btn:first-child:not(:last-child):not(.dropdown-toggle)` which has specifity of 0/0/5/0 while my rule has 0/0/2/0. I don't think it makes sense to add in an id selector, nor add 4 class/attribute selectors.